__tests__/integration/api.test.js

```diff
@@ -83,10 +83,30 @@ describe("/api/articles/:article_id", () => {
           created_at: expect.any(String),
           votes: expect.any(Number),
           article_img_url: expect.any(String),
+          comment_count: expect.any(Number),
```

Comment:
We should be checking this using a new test rather than altering previous ones 🧪